### ISSUE TITLE
Notification Center does not work???

### ISSUE THREAD

**Issue by Previesam** [SeptemberHX/dde-top-panel#26]
Well done team and thank you for this great creation. However i would like to know if it is intentional that the notification center not work when deepin dock is not running? I would really love to have that functionality so i can see all notifications even if i missed them.

**Comment by SeptemberHX** [comment]
Do you mean the notification plugin on the dde-dock with the bell icon ? If so, just copy /usr/lib/dde-dock/plugins/libnotifications.so to ~/.local/lib/dde-top-panel/plugins

**Comment by Previesam** [comment]
Yes. i have that in ~/.local/lib/dde-top-panel/plugins already. but unless the dde-dock itself is running the notification center does not work. i receive notification toast but when i click on the notification icon to open the notification center it simply does nothing.

**Comment by SeptemberHX** [comment]
Seems dde-dock does something important when it is launched, which will effect libnotification plugin. Further investigation is needed.

**Comment by Previesam** [comment]
Alright, thank you for your time.

**Comment by deejungx** [comment]
I am also having the same issue. Furthermore, as I'm using dual monitor, only the right half of the notification toast appears on my left monitor. I have tried changing my main monitor to see if that solves the problem but didn't work.

**Comment by SeptemberHX** [comment]
> I am also having the same issue. Furthermore, as I'm using dual monitor, only the right half of the notification toast appears on my left monitor. I have tried changing my main monitor to see if that solves the problem but didn't work.

Does this happen with dde-dock ?

**Comment by deejungx** [comment]
> Does this happen with dde-dock ?

No. I don't recall having the same issue with dde-dock. The position of the toast changed from bottom-right to top-left. I can't seem to find a way to change this behavior.
